docs/infra/api-markdown-documenter/render-api-documentation.mjs

```diff
@@ -141,13 +141,22 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 			}
 			return alerts;
 		},
-		skipPackage: (apiPackage) => {
-			const packageName = apiPackage.displayName;
-			const packageScope = PackageName.getScope(packageName);
+		exclude: (apiItem) => {
```

Comment:
`skipPackage` was generalized to `exclude` in v19. Allows more generic exclusion of API items (beyond just packages). Code updated to preserve existing package-skipping behavior (+ a couple of TODOs)